crates/sequencer/Cargo.toml

```diff
@@ -13,13 +13,16 @@ license.workspace = true
 # Starknet
 # TODO: remove the blockifier patch on the workspace once we can remove Katana.
 blockifier = { workspace = true }
+serde.workspace = { workspace = true }
+serde_json.workspace = true
```

Comment:
can we keep one syntax for the workspace?

Reply:
fixed via [this](https://github.com/kkrt-labs/ef-tests/pull/555/commits/b563ec5e17d1ffdc8663505a6880834dcdde40bb) commit.